dvc/repo/plots/__init__.py

```diff
@@ -309,9 +309,11 @@ def _collect_plots(
     )
 
     result = {
-        repo.dvcfs.from_os_path(plot.fs_path): _plot_props(plot) for plot in plots
+        os.path.relpath(plot.fs_path, repo.root_dir): _plot_props(plot)
```

Comment:
But root_dir might be a git path

Reply:
> But root_dir might be a git path

Not sure I follow.  In which case it could be? 

The current code is missing `os.path.isabs` check when compared to `from_os_path`, but the main point is to not convert to poxispath.

Reply:
> Not sure I follow. In which case it could be?

nvm I think I've found it in `plots diff`

Reply:
when within brancher, which doesn't apply here, but assuming `root_dir` is a local path and using os.path with it directly is dangerous. Better to use repo.fs then.